### ISSUE TITLE
Splitter splitbar and resizer not visible in high contrast mode

### ISSUE THREAD

**Issue by dimodi** [telerik/kendo-themes#4087]
**Describe the bug**

Reported in the [Blazor forums](https://www.telerik.com/forums/blazor-splitter-component-separator-has-accessibility-issues---separator-is-not-visible-in-aquatic-high-contrast-mode-or-in-desert-mode).

The Splitter splitbar and the resize handle rely only on a background color, which is lost in high contrast mode. We should use borders (at least for the resize handle), so that we preserve visibility.

Optionally, the border colors may match the background colors, so that the borders are not visible in standard mode.

**To reproduce**

[Enable High Contrast mode in the OS](https://support.microsoft.com/en-us/windows/turn-high-contrast-mode-on-or-off-in-windows-909e9d89-a0f9-a3a9-b993-7a6dcee85025) and then [go to any Splitter demo](https://demos.telerik.com/blazor-ui/splitter/orientation).

**Expected behavior**

Users should see the Splitbar (resizer).

**Affected package (please remove the unneeded items)**
* theme-default
* theme-bootstrap
* theme-material
* theme-tasks

**Affected suites (please remove the unneeded items)**
* Kendo UI for jQuery
* Kendo UI for Angular
* Kendo UI for React
* Kendo UI for Vue
* Telerik UI for Blazor

**Affected browsers (please remove the unneeded items)**
* All
